applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -180,6 +180,10 @@ config BT_OBSERVER
 	bool
 	default y
 
+config BT_PERIPHERAL
```

Comment:
I remember removing some of these configs since it was not necessary to change the default values. But i guess now something has changed making it necessary?

Reply:
Yes, they were needed for setting location for bis_headsets